src/pages/access_review/ApplicationSection.js

```diff
@@ -2,12 +2,24 @@ import React from 'react';
 import { div, hh } from 'react-hyperscript-helpers';
 import { Theme } from '../../libs/theme';
 
-const TEXT = {
-  fontSize: Theme.font.size.small,
-  lineHeight: Theme.font.leading.regular,
-};
+const highlightedWords = ["race", "ethnic", "ethnicity", "transethnic", "gender", "sex", "illegal", "illicit", "stigma",
+  "behavior", "drug", "alcohol", "addict", "religion", "religious", "intellect", "intelligence", "economic", "poor",
+  "poverty", "marginalized", "impoverished", "SES", "socioeconomic"];
 
 export const ApplicationSection = hh(class ApplicationSection extends React.PureComponent {
+
+
+  highlightExactMatches = (highlightedWords, content) => {
```

Comment:
Would love to see this as a utility function instead of something limited to `ApplicationSection`